pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -189,16 +189,27 @@ func (jm *ControllerV2) sync(ctx context.Context, cronJobKey string) (*time.Dura
 		return nil, err
 	}
 
-	cronJobCopy, requeueAfter, err := jm.syncCronJob(ctx, cronJob, jobsToBeReconciled)
+	updateStatus := false
+	cronJobCopy, requeueAfter, err := jm.syncCronJob(ctx, cronJob, jobsToBeReconciled, &updateStatus)
```

Comment:
better return the boolean instead of passing it as a pointer

Reply:
fixed